pkgs/games/prismlauncher/default.nix

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
File needs a `nixpkgs-fmt`

Reply:
Not sure if it's official/codified, I've been asked to use it in other PRs and remember seeing requests.

cc @SuperSandro2000 ?

Reply:
alejandra is doing a style uncomaptible with nixpkgs-fmt so it can't be used.

Reply:
I've switched over to nixpkgs-fmt